ibis/pandas/core.py

```diff
@@ -326,11 +353,11 @@ def execute_until_in_scope(
     # there should be exactly one dictionary per computable argument
     assert len(computable_args) == len(scopes)
 
-    new_scope = toolz.merge(new_scope, *scopes)
-
+    for s in scopes:
+        new_scope.merge_scope(s)
```

Comment:
Can you do
```
new_scope.merge_scopes(scopes)
```
Here?

Reply:
Sure, added another API of merge_scopes to unify these.